checkST2110.js

```diff
@@ -1621,8 +1715,9 @@ const allSections = (sdp, params) => {
       if (params.noCopy) {
         sections.push(no_copy_20);
       }
-    }
-    else {
+    } else if (mtParams[0]._encodingName == 'smpte291') {
+      sections = [section_40_2023_7];
```

Comment:
I think maybe we want at least `section_10_62, section_10_74, section_10_81, section_10_82, section_10_83, section_10_87` as well? Not sure if any of the -20/-21 test cases are applicable?

Reply:
Noted. 

A couple of additions for -40 regarding `exactframerate` and `TROFF`.

- `exactframerate` is now mandatory. I thought about simply adding `test_20_72_3` to the Section function, but that doesn't enforce the parameter, - I could have modified `test_20_72_3` to add a -40 special case in, but instead opted to factor out the logic and create a new test: `test_40_2023_7_3`. 
- 'TROFF' I did something similar, but I probably could have just added `test_21_82_1` to the Section function - however that test bypasses if audio only, so wasn't sure if that was a safe assumption for -40 too.

Thoughts?

Reply:
I like this refactoring, yeah! IIRC, `exactframerate` isn't mentioned in previous revision of -40 at all, so it makes sense not to even look for it?

Reply:
Regarding TROFF, I think the refactoring is still good, especially if -40 doesn't need any of the other -21 tests?

I suspect that the use of `audioOnly` in the `test_21_82_x` functions may now be unnecessary because that section is not included in `allSections` when the media type is audio?

Reply:
Removed `audioOnly` checks in the -21 tests it is, as you correctly stated, redundant.